### DISCUSSION TITLE
BestBuy, BestBuy-CA, Amazon-CA not working?

### DISCUSSION THREAD

**Discussion by Sazerak** [jef/streetmerchant#3145] [Help]
I've been using streetmerchant for a few days now and noticed that I wasn't getting any hits from many of the stores I was trying. I've been trying 5 stores (target, walmart-ca, amazon-ca, bestbuy, bestbuy-ca) and noticed that only target and walmart-ca were showing in stock properly.

I tried changing the products I was looking for just in case and tried changing the container/text values to a few different things I could find while inspecting the button, but no luck. Can anyone confirm if theirs is working and/or how to get it working if it isn't?

**Comment by Mr-Tech-13** [comment]
Is it possible that the products you are looking for are out of stock?

**Reply by Mr-Tech-13** [reply]
If not, make HEADLESS=false in dotenv and see what streetmerchant sees
